gameroom.js

```diff
@@ -182,6 +184,8 @@ class GameRoom {
         player2_name: this.players[1].name,
         player2_id: this.players[1].id,
         player2_deck_id: this.players[1].deck_id,
+        starting_timer: this.starting_timer,
```

Comment:
Please add these to the
const matchData = {
as well.
Everything in that dictionary gets saved in the game log when it completes.